tests/es6-super.js

```diff
@@ -29,7 +29,7 @@ const analyze = require("..").analyze;
 describe("ES6 super", () => {
     it("is not handled as reference", () => {
         const ast = espree(`
-            class Hello {
+            class Foo extends Bar {
```

Comment:
More recent versions of Espree will throw an error when super methods are called in methods of a class that is not a subclass: `SyntaxError: super() call outside constructor of a subclass`